Source/init.cpp

```diff
@@ -94,7 +94,7 @@ HANDLE init_test_access(char *mpq_path, char *mpq_name, char *reg_loc, int dwPri
 
 	for (int i = 0; i < 2; i++) {
 		snprintf(mpq_path, MAX_PATH, "%s%s", Buffer[i], mpq_name);
-#ifndef __SWITCH__
+#ifdef __SWITCH__

```

Comment:
This will probably break Nintendo Switch support.

Reply:
This should not be necessary, as it was restored already.
See: https://github.com/diasurgical/devilutionX/commit/4c96d946c6d1cf14cb9f1e976ec47259fdee114a#diff-28e77940c81824301914e92db3a86793